introduction.md

```diff
@@ -0,0 +1 @@
+This is a repo for a course I am doing with codeschool.com
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
Mine would have to be git, python then the electives. I am more of an embedded system or C++ general programmer than web interface...

Reply:
Oh in the file...duh